src/mobile/apps/artwork/components/meta_data/templates/index.jade

```diff
@@ -7,8 +7,6 @@
       include ./edition_sets.jade
       include ./inquiry.jade
 
-    include ./partner.jade
```

Comment:
Oh yea, is this nixing that file from above, with the empty comment?

Reply:
It's actually nixing this file:
![image](https://user-images.githubusercontent.com/2081340/47436531-a4735480-d774-11e8-9e2b-135969f5e323.png)